.github/workflows/release-embedding-sdk.yml

```diff
@@ -322,6 +322,7 @@ jobs:
       - name: Check out branch to prepare for a SDK version bump PR
         uses: actions/checkout@v4
         with:
+          token: ${{ secrets.METABASE_AUTOMATION_USER_TOKEN }}
```

Comment:
This is the heart of the fix. [When specifying `token`](https://github.com/actions/checkout?tab=readme-ov-file#usage), it will be used with all git commands.

We repeat this step when pushing tags as well since it's a separate job.